packages/augur-ui/webpack.config.js

```diff
@@ -15,7 +15,7 @@ const DeadCodePlugin = require("webpack-deadcode-plugin");
 // DEVELOPMENT CONFIG
 if (!process.env.DEBUG_BUILD && process.env.NODE_ENV === "development") {
   baseConfig = merge(baseConfig, {
-    // devtool: "eval-source-map",
+    devtool: "eval-source-map",
```

Comment:
I think this causes dev process to really slow down. prob don't want to check this in.